lib/src/integrationTest/kotlin/com/lemonappdev/konsist/core/declaration/koclassdeclaration/KoClassDeclarationForNameTest.kt

```diff
@@ -0,0 +1,20 @@
+package com.lemonappdev.konsist.core.declaration.koclassdeclaration
+
+import com.lemonappdev.konsist.TestSnippetProvider.getSnippetKoScope
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.jupiter.api.Test
+
+class KoClassDeclarationForNameTest {
```

Comment:
It looks like we don't need this test, because KoClassDeclarationImpl does nto define name property - this test is/shouldbe covered by the KoNamedDeclarationImpl